lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -125,6 +126,7 @@
         slice(array, start, end) { return callInstanceFunc(platform.builtInJavascriptArrayEntrySlice, array, start, end); },
         concat(array, ...els) { return callInstanceFunc(platform.builtInJavascriptArrayEntryConcat, array, ...els); },
         filter(array, func) { return callInstanceFunc(platform.builtInJavascriptArrayEntryFilter, array, func); },
+        fill(array, value) { return callInstanceFunc(platform.builtInJavascriptArrayEntryFill, value); },
```

Comment:
This should use the passed in `array` right?

Reply:
oh, wow, yeah. good catch!